src/sfnt/ttcolr.c

```diff
@@ -345,22 +345,32 @@
   }
 
   static FT_Bool
-  read_paint ( Colr *         colr,
-               FT_Byte *      layer_v1_array,
-               FT_ULong       paint_offset,
-               FT_COLR_Paint *apaint )
+  read_paint( Colr* colr, FT_Byte* p, FT_COLR_Paint* apaint )
```

Comment:
I *hate* the clang formatter for doing this!  There is no possibility to configure it to do one variable per line even if the line is short.  Needless to say that FreeType needs one line per variable...